packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3021,6 +3022,7 @@ export default class Meeting extends StatelessWebexPlugin {
         webexMeetingInfo?.hostId ||
         this.owner;
       this.permissionToken = webexMeetingInfo?.permissionToken;
+      this.permissionTokenTtl = webexMeetingInfo?.permissionTokenTtl;
```

Comment:
suggestion: For whatever reason, the `permissionTokenTtl` field is actually supplied as a string from the API:

![Screenshot 2023-11-13 at 15 15 23](https://github.com/webex/webex-js-sdk/assets/82158095/c6caf0b2-7086-4f6f-a09b-4f49b51e55b1)

Not sure if this is a mistake on the service side (feels like it is), but we may want to ensure it's a number so that we can rely on the type from this point onwards. I realise the calculation will happen to work because js is overly permissive, but it would be nice to be able to rely on it.